src/constants/common.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
 		'/sessions/removeAllSessions',
 		'/mentoring/v1/users/add',
 		'/mentoring/v1/users/delete',
+		'/requestSessions/expire',
```

Comment:
@rakeshSgr where is the endpoint called from what workflow?

Reply:
When session requested created, creating a job in scheduler, so that once session requested is expired, it should call the this API to set that status as exipred